src/collection.ts

```diff
@@ -192,6 +192,13 @@ export class Collection<TSchema extends Document = Document> {
     this.client = db.client;
   }
 
+  /**
+   * Get the database object for the collection.
+   */
+  get db(): Db {
+    return this.s.db;
+  }
+
```

Comment:
Thoughts on also making this a readonly property on the class (like how `client` is defined on the db)?

```suggestion
  /**
   * Get the database object for the collection.
   */
   readonly db: Db;
```

We've talked about removing the `s` properties for a while now, this would be one fewer property we're storing on `s` 🙂

Reply:
Done.